src/core/client/admin/routeConfig.tsx

```diff
@@ -8,27 +8,27 @@ import { Ability } from "./permissions";
 import { createAuthCheckRoute } from "./routes/AuthCheck";
 import CommunityRoute from "./routes/Community";
 import ConfigureRoute from "./routes/Configure";
+import InnerFormLayout from "./routes/Configure/InnerFormLayout";
```

Comment:
Refactored to use this new `InnerFormLayout` for all cases similar to adding/editing domains (moderation phases, webhook endpoints, sites adding/editing), since they were all using the same layout already.